service/src/intTest/resources/application.yml

```diff
@@ -43,7 +43,7 @@ gp2gp:
     password: ${GP2GP_MONGO_PASSWORD:}
     options: ${GP2GP_MONGO_OPTIONS:}
     autoIndexCreation: ${GP2GP_MONGO_AUTO_INDEX_CREATION:true}
-    ttl: ${GP2GP_MONGO_TTL:P7D}
+    ttl: ${GP2GP_MONGO_TTL:P12W}
```

Comment:
I think P12W is not the correct format, P84D should be better and not to cause any test failures